packages/babel-traverse/src/scope/index.ts

```diff
@@ -369,7 +369,7 @@ const collectorVisitor: Visitor<CollectVisitorState> = {
       // @ts-expect-error Fixme: document symbol ast properties
       !path.get("id").node[NOT_LOCAL_BINDING]
     ) {
-      path.scope.registerBinding("local", path);
+      path.scope.registerBinding("local", path.get("id"), path);
```

Comment:
Since the changes in `@babel/types` will break old versions of `@babel/traverse`, here we defer the `ClassExpression` change to Babel 8. On the other hand, the `UpdateExpression` change does not break other packages, so we could ship it in Babel 7.